examples/plot_2_multidim_sigs.py

```diff
@@ -0,0 +1,249 @@
+"""
+2. 2D and 3D Signals
+====================
+Bycycle supports computing features for 2D and 3D arrays of signals, using
+:func:`~.compute_features_2d` or :func:`~.compute_features_3d`. These signals may be organized in a
+variety of ways, such as (n_epochs, n_timepoints), (n_channels, n_timepoints), or
+(n_channels,  n_epochs, n_timepoints). The ``axis`` keyword argument may be used to handle various
+organizations of signals. Understanding the axis argument is important to ensure features are
+computed in an appropriate manor.
+
+For 2D arrays:
```

Comment:
Having these axis descriptions here feels like data dump, with no clear way to use this information yet. Each respective axis description should be moved to be with respective examples below that do 2D / 3D examples.